### DISCUSSION TITLE
电子书繁体竖排转换简体横排 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#39] [Announcements]
# 电子书繁体竖排转换简体横排 - forecho's Blog

引言
我现在有一本繁体竖排的电子书，竖排真的是没那么阅读习惯。繁体看着也累，于是想到了做个转换，于是网上搜索资料，但搜索到的都是与我反过来的，他们要把简体横排转换成繁体竖排。
于是我只能自己摸索着如何把『电子书繁体竖排转换简体横排』，然后整理了成这篇文章。

https://blog.forecho.com/e-book-traditional-vertical-conversion-simplified-horizontal.html

**Comment by vvzkk** [comment]
太棒了! 感谢分享!

**Comment by CrippledRider** [comment]
你好，搜索不到这个插件

**Reply by Chandler421** [reply]
装新版的calibre

**Reply by forecho** [reply]
插件的介绍链接 https://www.mobileread.com/forums/showthread.php?t=275572

**Comment by duharry0915** [comment]
好的，非常感谢🙏，很有用！

**Comment by ZebinGao** [comment]
图片格式的竖排繁体能转换成横排简体吗？

**Reply by forecho** [reply]
图片肯定不行了，但是你可以借助 AI ，让它帮忙转。我有一个中转站，里面可以用图片模型，你可以试试 https://4aicode.com/
